ci_scripts/build_macos.sh

```diff
@@ -135,3 +145,8 @@ cmake --build . --config Release -- -sdk iphoneos CODE_SIGN_IDENTITY="" CODE_SIG
 # echo "Build KTX-Software (iOS Simulator Release)"
 # cmake --build . --config Release -- -sdk iphonesimulator
 popd
+
+# Java
+
+LIBKTX_BINARY_DIR=$(pwd)/$DEPLOY_BUILD_DIR/Debug ci_scripts/build_java.sh
```

Comment:
Why choose the Debug configuration for building the Java wrapper?

Reply:
I don't know. I haven't worked with Xcode before. If there are trivial configuration changes desired, please modify freely.

Reply:
This doesn't have anything to do with Xcode. When `build_java.sh` runs and Maven builds the Java bindings does it make any difference to the result whether the libktx_jni used is Debug or Release? If it does, I think we should use the Release configuration.

Reply:
```suggestion
LIBKTX_BINARY_DIR=$(pwd)/$DEPLOY_BUILD_DIR/Release ci_scripts/build_java.sh
```

Reply:
I changed it to Release.